fhir-core/src/test/java/com/ibm/fhir/core/test/TimestampTest.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * (C) Copyright IBM Corp. 2016,2019
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.core.test;
+
+import static org.testng.Assert.assertTrue;
+
+import java.util.Date;
+
+import org.testng.annotations.Test;
+
+import com.ibm.fhir.core.FHIRUtilities;
+
+public class TimestampTest {
+    
+    @Test
+    public void test1() throws Exception {
+        
+        String strTimeStamp = FHIRUtilities.formatTimestamp(new Date(System.currentTimeMillis()));
```

Comment:
you could just use `new Date()`...it creates it from System.currentTimeMillis() by default
```suggestion
        String strTimeStamp = FHIRUtilities.formatTimestamp(new Date());
```

Reply:
done! I didn't commit your proposal but made the change in my local directly and push. because otherwise, I will have to fix the DCO issue again.